index.html

```diff
@@ -63,7 +86,7 @@ <h3 class="text_titleH3">Project 1</h3>
             <div class="container-work" id="project2">
                 <div class="img-work img-work-source2"></div>
                 <div class="div-title flex-work-title">
-                    <h3 class="text_titleH3">Project 2</h3>
+                    <h3 class="text-titleH3">Project 2</h3>
                 </div>
                 <div class="flex-ul-work">
                     <div class="ul-work-item">
```

Comment:
[OPTIONAL]

Kindly make sure to make this project a little smaller, it should take closer to 25% of the page's width